content/guides/models/registry/configure_registry.md

```diff
@@ -7,107 +7,121 @@ title: Configure registry access
 weight: 3
 ---
 
-<!-- A registry, and the linked artifacts inside a registry, belong to an organization. This means that teams within an organization can publish and consume artifacts linked to a registry, if that team has correct access control. -->
+Registry admins can [configure registry roles]({{< relref "configure_registry.md#configure-registry-roles" >}}), [add users]({{< relref "configure_registry.md#add-a-user-or-a-team-to-a-registry" >}}), or [remove users]({{< relref "configure_registry.md#remove-a-user-or-team-from-a-registry" >}}) from a registry by configuring the registry's settings.
 
-Registry admins can limit who can access a registry by navigating to a registry's settings and assigning a user's role to [Admin, Member, or Viewer]({{< relref "#registry-roles-permissions" >}}). Users can have different roles in different registries. For example, a user can have a view role in "Registry A" and a member role in the "Registry B". 
+## Add a user or a team to a registry
+
+Registry admins can add individual users or entire teams to a registry. To add a user or team to a registry:
+
+1. Navigate to the Registry App at https://wandb.ai/registry/.
```

Comment:
Can we just say "Navigate to <IURL>" without referring to it as an App? Maybe that's a convention.